flybywiresim/aircraft
# fix(EFB): Agnostic opt in to error reporting

Fixes #9232

## Summary of Changes

Change error reporting popup to be plane agnostic rather than just A32NX


## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): Yahtzee94

## Testing instructions

1. Launch either aircraft at stand or runway
2. Power on EFB
3. In settings go to ATSU/AOC
4. Toggle errror reporting on
5. Confirm correct message appears


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
